kbdcfg.lua

```diff
@@ -130,6 +130,9 @@ local function factory(args)
     kbdcfg.current               = args.current or kbdcfg.default_layout_index
     kbdcfg.menu                  = nil
     kbdcfg.type                  = args.type or "tui"
+    kbdcfg.tui_wrap              = args.tui_wrap or " "
```

Comment:
I have some doubts about this variable. Do you think that we really need it? Probably it is enough to give only one option for the user (to specify symbol for left and right wrappings separately). Too many options can confuse. When the user wants to configure the wrapping, I don't think that it will be a problem to initialize two variables instead of one.

Reply:
My thinking was that most users would probably just want to change ``tui_wrap`` and not both settings separately, but I suppose you're right.